Dockerfile

```diff
@@ -1,30 +1,37 @@
-FROM rust:1.27 AS librespot
+FROM rust:1.42 AS librespot
 
 RUN apt-get update \
  && apt-get -y install build-essential portaudio19-dev curl unzip \
  && apt-get clean && rm -fR /var/lib/apt/lists
 
+ARG LIBRESPOT_VERSION=0.1.1
+
 RUN cd /tmp \
- && curl -sLO https://github.com/librespot-org/librespot/archive/master.zip \
- && unzip master.zip \
- && cd librespot-master \
+ && curl -sLO https://github.com/librespot-org/librespot/archive/v0.1.1.zip \
+ && unzip v${LIBRESPOT_VERSION}.zip \
+ && cd librespot-${LIBRESPOT_VERSION} \
  && cargo build --release \
- && chmod +x /tmp/librespot-master/target/release/librespot
+ && chmod +x target/release/librespot
 
-FROM ubuntu
+FROM ubuntu:bionic
 
 RUN apt-get update \
- && apt-get -y install curl libportaudio2 libvorbis0a libavahi-client3 libflac8 libvorbisenc2 libvorbisfile3 \
+ && apt-get -y install curl libportaudio2 libvorbis0a libavahi-client3 libflac8 libvorbisenc2 libvorbisfile3 libopus0 \
  && apt-get clean && rm -fR /var/lib/apt/lists
 
 ARG ARCH=amd64
-ARG SNAPCAST_VERSION=0.11.1
+ARG SNAPCAST_VERSION=0.19.0
+ARG LIBRESPOT_VERSION=0.1.1
```

Comment:
This is a duplicate from above.

Reply:
It must be duplicated as the second part is a second build. Unfortunate, and maybe there is a better way.